public/resources/remix/style.less

```diff
@@ -89,17 +89,19 @@
   &.mouse-mode {
 
     width: 400px !important;
-    top: 5px !important;
+    top: 6px !important;
     right: 6px !important;
-    height: 64px !important;
-    background: white !important;
+    height: 75px !important;
+    background: ghostwhite !important;
     padding: 11px 110px 11px 76px !important;
     box-sizing: border-box !important;
     font-family: "Fira Medium", "Helvetica Neue", sans-serif !important;
     box-shadow: 0 2px 2px rgba(0,0,0,.5) !important;
```

Comment:
Hey @JohnEJames - just get rid of this line (the box-shadow) and we'll be good to go!

Thanks for all of this work and for your patience. I know it's annoying to resolve all of these small styling issues commit by commit in this fashion, but I'm glad we're almost there.